frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -294,15 +294,18 @@ function ReaderCoptListener:getAltStatusBarMenu()
             },
             {
                 text_func = function()
-                    local status = _("off")
+                    local status = _(" ")
                     if self.battery == 1 then
                         if self.battery_percent == 1 then
-                            status = _("percentage")
+                            status = _("(percentage)")
```

Comment:
This goes against our style guide (i.e., with a colon and not with parentheses), but as a more generic point you shouldn't bother translators with them.